### ISSUE TITLE
Unexpected file in index: .DS_Store.

### ISSUE THREAD

**Issue by Blady-Com** [alire-project/alire#1059]
It seems that macOS .DS_Store file puzzles alire:
error: Unexpected file in index: /Users/me/.config/alire/indexes/community/repo/index/gn/.DS_Store
% alr --version
alr 1.2.0

**Comment by mosteo** [comment]
Right, it will protest because of extraneous files. How/when is that file generated? Other macOS users haven't reported any problems.

**Comment by Blady-Com** [comment]
[DS_Store](https://en.wikipedia.org/wiki/.DS_Store) file is created to store custom attributes using the Finder (GUI macOS file explorer).

**Comment by simonjwright** [comment]
I guess most of us use the command line! (I have to remove emacs backup files, foo.toml~)

**Comment by mosteo** [comment]
OK, this can be downgraded to a warning or note for regular users, keeping the error only for the `--strict` checks.
